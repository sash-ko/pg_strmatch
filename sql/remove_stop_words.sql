CREATE OR REPLACE FUNCTION remove_stop_words(text) RETURNS text
    AS 'pg_strmatch.so', 'remove_stop_words'
    LANGUAGE C STRICT;

SELECT remove_stop_words(NULL) IS NULL;
SELECT remove_stop_words('') = '';
SELECT remove_stop_words('  ') = '';
SELECT remove_stop_words('the') = '';
SELECT remove_stop_words('test of the test') = 'test test';
SELECT remove_stop_words('test') = 'test';
