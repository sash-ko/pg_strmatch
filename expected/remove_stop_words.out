CREATE OR REPLACE FUNCTION remove_stop_words(text) RETURNS text
    AS 'pg_strmatch.so', 'remove_stop_words'
    LANGUAGE C STRICT;
SELECT remove_stop_words(NULL) IS NULL;
 ?column? 
----------
 t
(1 row)

SELECT remove_stop_words('') = '';
 ?column? 
----------
 t
(1 row)

SELECT remove_stop_words('  ') = '';
 ?column? 
----------
 t
(1 row)

SELECT remove_stop_words('the') = '';
 ?column? 
----------
 t
(1 row)

SELECT remove_stop_words('test of the test') = 'test test';
 ?column? 
----------
 t
(1 row)

SELECT remove_stop_words('test') = 'test';
 ?column? 
----------
 t
(1 row)

